### ISSUE TITLE
Windows 10 ARM DLL ?

### ISSUE THREAD

**Issue by YT-Advanced** [cinit/WSAPatch#32]
Can you update ARM64 DLL (include icu.dll, winhttp.dll and WSAPatch.dll). Thanks

**Comment by MustardChef** [comment]
I can provide the original arm64 DLLs if needed and also test WSA on Windows 10 arm

**Comment by YT-Advanced** [comment]
> I can provide the original arm64 DLLs if needed and also test WSA on Windows 10 arm

Yes, but I don't have any Surface or another ARM Device for testing

**Comment by MustardChef** [comment]
> > I can provide the original arm64 DLLs if needed and also test WSA on Windows 10 arm
> 
> Yes, but I don't have any Surface or another ARM Device for testing

I have an arm device to test on

**Comment by YT-Advanced** [comment]
> > > I can provide the original arm64 DLLs if needed and also test WSA on Windows 10 arm
> > 
> > Yes, but I don't have any Surface or another ARM Device for testing
> 
> I have an arm device to test on

So can you build ARM64 DLLs and test it now ?
